spring-cloud-dataflow-configuration-metadata/pom.xml

```diff
@@ -49,7 +49,7 @@
 		</dependency>
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-loader</artifactId>
+			<artifactId>spring-boot-loader-classic</artifactId>
```

Comment:
What is the significance of this?

Reply:
`org.springframework.boot.loader.archive.JarFileArchive` (The one dataflow uses) moved to this dependency.   There is another `org.springframework.boot.loader.launch.JarFileArchive`.  I chose the one that dataflow has used in the past.